pkg/providers/aws/cluster_network_provider.go

```diff
@@ -26,77 +38,562 @@ import (
-	/*
-		todo CreateNetwork is called we need to ensure the following
-		we expect _network to exist with a valid cidr
-		the absence of either _network or valid cidr we should return with an informative message
-		re-reconcile until we have a valid _network strat and valid cidr
+/*
+createStandaloneVPC returns a Network type and error
+
+VPCs must be created with a valid CIDR Block range, between \16 and \26
+if an invalid range is passed, the function returns an error
+
+VPCs are tagged with the name `<organizationTag>/clusterID`.
+By default, `integreatly.org/clusterID`.
 
-		- Check if a VPC exists with the integreatly.org/clusterID tag
-		- If VPC doesn't exist, create a VPC with CIDR block and tag it
-	*/
-	logger := n.logger.WithField("action", "CreateNetwork")
-	logger.Debug("CreateNetwork stub")
-	return errorUtil.New("CreateNetwork stub")
+NOTE - Once a VPC is created we do not want to update it. To avoid changing cidr block
+*/
+func (n *NetworkProvider) createStandaloneVpc(vpcCidrBlock *net.IPNet, clusterID, organizationTag string) (*Network, error) {
+	logger := n.Logger.WithField("action", "creating standalone vpc")
+	// expected valid CIDR block between /16 and /26
+	if !isValidCIDRRange(vpcCidrBlock) {
+		return nil, errorUtil.New(fmt.Sprintf("%s is out of range, block sizes must be between `/16` and `/26`, please update `_network` strategy", vpcCidrBlock.String()))
+	}
+
+	logger.Infof("cidr %s is valid 👍", vpcCidrBlock.String())
+
+	// create vpc using cidr string from _network
+	createVpcOutput, err := n.Ec2Api.CreateVpc(&ec2.CreateVpcInput{
+		CidrBlock: aws.String(vpcCidrBlock.String()),
+	})
+	ec2Err, isAwsErr := err.(awserr.Error)
+	if err != nil && isAwsErr && ec2Err.Code() == "InvalidVpc.Range" {
+		return nil, errorUtil.New(fmt.Sprintf("%s is out of range, block sizes must be between `/16` and `/26`, please update `_network` strategy", vpcCidrBlock.String()))
+	}
+	if err != nil {
+		return nil, errorUtil.Wrap(err, "unexpected error creating vpc")
+	}
+	logger.Infof("creating vpc: %s for clusterID: %s", *createVpcOutput.Vpc.VpcId, clusterID)
+
+	// ensure standalone vpc has correct tags
+	clusterIDTag := &ec2.Tag{
+		Key:   aws.String(fmt.Sprintf("%sclusterID", organizationTag)),
+		Value: aws.String(clusterID),
+	}
+	if err = n.reconcileVPCTags(createVpcOutput.Vpc, clusterIDTag); err != nil {
+		return nil, errorUtil.Wrapf(err, "unexpected error while reconciling vpc tags")
+	}
+
+	return &Network{
+		Vpc: createVpcOutput.Vpc,
+	}, nil
 }
 
-func (n *NetworkProvider) DeleteNetwork() error {
-	logger := n.logger.WithField("action", "DeleteNetwork")
-	logger.Debug("DeleteNetwork stub")
+/*
+DeleteNetwork returns an error
+
+VPCs are tagged with with the name `<organizationTag>/clusterID`.
+By default, `integreatly.org/clusterID`.
+
+This tag is used to find a standalone VPC
+If found DeleteNetwork will attempt to remove:
+	* all vpc associated subnets
+	* both subnet groups (rds and elasticache)
+	* the vpc
+*/
```

Comment:
It looks like there's a mix of block comment formatting used. I've been using `//` in the past as it's what Goland defaults to, on macOS at least. I don't have a preference but we should probably stick with one